tests/test_pygtracker.py

```diff
@@ -22,6 +33,34 @@ def test_version():
 
 # Start tests for rank_students
 
+# tests that the user inputs the right number of students
+def test_number_of_students():
```

Comment:
function name should be

test_functiontotest_conditionthatyoutest
```suggestion
def test_ranks_students_num_of_students_match():
```